app/containers/message/Urls.tsx

```diff
@@ -52,15 +53,21 @@ const styles = StyleSheet.create({
 });
 
 const UrlImage = React.memo(
-	({ image }: { image: string }) => {
+	({ image, hasContent }: { image: string; hasContent: boolean }) => {
```

Comment:
This is awkward.
We already have a param `image` we check to understand if there's content or not.
You're adding yet another one to render a border radius?
It looks very suspicious...

You could merge the components.